packages/engine-server/src/workspace/service.ts

```diff
@@ -469,6 +473,37 @@ export class WorkspaceService {
     return ws;
   }
 
+  static async createEmptyWorkspace(opts: WorkspaceServiceCreateOpts) {
+    const { wsRoot, vaults } = opts;
+    const ws = new WorkspaceService({ wsRoot });
+    fs.ensureDirSync(wsRoot);
+    // this creates `dendron.yml`
+    ws.createConfig();
+    // add gitignore
+    const gitIgnore = path.join(wsRoot, ".gitignore");
```

Comment:
- can we extract writing the ignore contents into a separate function and add `seeds` to one of the ignored files?
- we can also use this in `createWorkspace`

Reply:
Apologies - this function `createEmptyWorkspace()` was just a temporary artifact I put here while prototyping; I forgot to delete before the review and it's not being used anywhere.